make_interleaved_WDL/make_interleaved_reads.wdl

```diff
@@ -21,24 +28,25 @@ workflow make_interleaved_reads {
 }
 
 task interleave_reads{
-
+	input{
 	Array[File] input_files
 	String output_file = "interleaved.fastq.gz"
-	
+	String container
+	}
 	command <<<
-	    if file --mime -b ${input_files[0]} | grep gzip; then 
-			paste <(gunzip -c ${input_files[0]} | paste - - - -) <(gunzip -c ${input_files[1]} | paste - - - -) | tr '\t' '\n' | gzip -c > ${output_file}
+		if file --mime -b ~{input_files[0]} | grep gzip; then 
```

Comment:
Please review all the tasks in the wdl to make sure pipeline is set if there is more than one bash executable so errors are caught by cromwell. 
command is 
'set -euo pipefail'